src/modules/commander/PreflightCheck.cpp

```diff
@@ -555,7 +555,7 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
 
 	if (status.pos_test_ratio > test_limit) {
 		if (report_fail) {
-			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Horizontal estimate Pos Error");
+			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: Horizontal position estimate Error");
```

Comment:
This is too long by 1 character (50 chars).

Reply:
@bkueng is the limit 50 or 49 chars?  I was under the impression 50 was okay.

Reply:
50 but the last one is a null-char.

Reply:
@bkueng  fixed here: https://github.com/PX4/Firmware/pull/11491

Reply:
FWIW I find the inconsistent capitalisation disturbing

"Preflight Fail: Horizontal position estimate Error" might be better as

> Preflight Fail: Horizontal Position Estimate Error
>  Preflight Fail: Horizontal position estimate error
>  Preflight Fail: Error - Horizontal position estimate

Reply:
@hamishwillee you are right, the mixed case is bad.  I've fixed it to lower case "error" in this updated PR:  https://github.com/PX4/Firmware/pull/11491